experiments.py

```diff
@@ -80,11 +83,15 @@ def print_scores(self):
         for game in self.global_score:
             player1 = game['first_player']
             player2 = game['second_player']
-            print(f"{player1['player_type']}  {player1['score']}:{player2['score']}  {player2['player_type']}")
+            print(f"Scores: {player1['player_type']}  {player1['score']}:{player2['score']}  {player2['player_type']}")
+            print(f"Wins: {player1['player_type']}  {player1['wins']}:{player2['wins']}  {player2['player_type']}")
+            print(f"Draws: {player1['player_type']}  {player1['draws']}:{player2['draws']}  {player2['player_type']}")
+            print(f"Loses: {player1['player_type']}  {player1['loses']}:{player2['loses']}  {player2['player_type']}")
```

Comment:
please add two empty lines to separate results from other games

Reply:
Sure, done!